packages/salesforcedx-vscode-apex/src/requirements.ts

```diff
@@ -102,14 +102,11 @@ const isLocal = (javaHome: string): boolean => {
 export const checkJavaVersion = async (javaHome: string): Promise<boolean> => {
   return new Promise((resolve, reject) => {
     cp.execFile(
-      javaHome + '/bin/java',
-      ['-version'],
+      javaHome + '/bin/javax',
+      ['-XshowSettings:properties', '-version'],
```

Comment:
Adding this property to the command will cause java to emit to stderr all of the System properties set in the VM, including `java.version`. I kept the `-version` option so that java does not emit the default help text.